src/EditorFeatures/CSharpTest/Diagnostics/Nullable/CSharpPossiblyDeclareAsNullableCodeFixTests.cs

```diff
@@ -0,0 +1,628 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
```

Comment:
"possibly declare as nullable" is an odd name...

Reply:
[Copied from elsewhere] Here are my thoughts on tooling for nullability:

I see three categories of situations for making a type nullable:
1) assignments of literal null (for example, `return null;` offers to fix the return type. `DeclareAsNullable` fixer already shipped in VS)
2) null tests (present PR), such as `x is null` or `x?.Method()`.
3) assignments of maybe-null values (not implemented). For example, `return maybeNull;` would offer to fix the return type. This is similar to [1] but should be separate in my opinion, because I would likely want to separate the FixAll for [1] from that for [3].

Later on, we may want to also discuss fixers to adjust an override to match the overridden definition, or vice-versa.

----

In that context, we already have [1] (DeclareAsNullable), and this PR implements a chunk of [2]. I'm not sure what to call it. The main point is that the likelihood that you want this fix is lower than that of `DeclareAsNullable` which is high-confidence ;-)